components/DropdownLight/DropdownLight.jsx

```diff
@@ -41,8 +48,16 @@ const ArrowIcon = styled(Icon)`
   width: 24px;
 `;
 
-const Wrapper = styled.div`
-  position: relative;
+const ComponentWrapper = styled.div`
+  ${({
+    theme: {
+      colors: { neutral },
+    },
+    skin = 'default',
```

Comment:
Is this right?

Reply:
![image](https://user-images.githubusercontent.com/3389749/220738095-9a83783c-dd1f-4507-b715-0bfea9c4e1f5.png)
![image](https://user-images.githubusercontent.com/3389749/220738226-c96c04a2-8b9c-4a67-b55a-e4b82b0356f1.png)

Reply:
When it happens? I simulated several scenarios to force this bug and nothing happened

Reply:
When i choose the skin `Dark`

Reply:
And look at the code.